src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/UdtTest/UdtTest2.cs

```diff
@@ -6,11 +6,6 @@
 using System.Data;
 using System.Data.SqlTypes;
 using System.Text;
-#if NETFRAMEWORK
-using Microsoft.SqlServer.Server;
```

Comment:
Why not keep this one using instead of adding the more specific references down below (lines 571-582)?

Reply:
It remained from the process of finding conflict on Linux. Done.